docs/Tutorials/Privacy/Configuring-Multi-Tenancy.md

```diff
@@ -0,0 +1,112 @@
+
+In the `Node-1/Orion` directory, [generate a public/private key pair for each
+tenant](Configuring-Privacy.md#3-generate-orion-keys).
+
+Name the key pair `nodeKey2` and `nodeKey3`.
+
+## 3. Update Password File
+
+Update `passwordFile` in the `Node-1/Orion` directory by adding each password
+used to generate the Orion keys on a new line.
+
+A separate password is required for each key pair, even if the passwords are
+identical.
+
+## 4. Update the Orion Configuration File
+
+In the `Node-1/Orion` directory, update the `orion.conf` file by adding the
+new key pairs:
+
+```bash
+nodeurl = "http://127.0.0.1:8080/"
+nodeport = 8080
+clienturl = "http://127.0.0.1:8888/"
+clientport = 8888
+publickeys = ["nodeKey.pub", "nodeKey2.pub", "nodeKey3.pub"]
+privatekeys = ["nodeKey.key", "nodeKey2.key", "nodeKey3.key"]
+passwords = "passwordFile"
+tls = "off"
+```
+
+## 5. Start Orion Nodes
+
+In each `Orion` directory, [start Orion](Configuring-Privacy.md#5-start-orion-nodes) and specify the configuration file.
+
+## 6. Start Besu Node-1
+
+In the `Node-1` directory, start Besu Node-1:
+
+```bash tab="MacOS"
+besu --data-path=data --genesis-file=../genesis.json --rpc-http-authentication-enabled --rpc-http-authentication-jwt-public-key-file=publicKey.pem --rpc-http-enabled --rpc-http-api=ETH,NET,IBFT,EEA,PRIV --host-whitelist="*" --rpc-http-cors-origins="all" --privacy-enabled --privacy-url=http://127.0.0.1:8888 --privacy-multi-tenancy-enabled --min-gas-price=0   
+```
+
+The command line specifies privacy options: 
+
+* [`--rpc-http-authentication-enabled`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-enabled) enables authentication
+for JSON-RPC APIs.
+* [`--rpc-http-authentication-jwt-public-key-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-jwt-public-key-file) specifies the Operator's [public key file](#1-generate-a-private-and-public-key-pair). Used to authenticate the [tenant JWT tokens](#6-generate-tenant-jwt-tokens).
+* [`--privacy-enabled`](../../Reference/CLI/CLI-Syntax.md#privacy-enabled) enables privacy.
+* [`--privacy-url`](../../Reference/CLI/CLI-Syntax.md#privacy-url) specifies the Orion node URL (`clienturl` in `orion.conf`)
+* [`--privacy-multi-tenancy-enabled`](../../Reference/CLI/CLI-Syntax.md#privacy-multi-tenancy-enabled) enables multi-tenancy.
+
+!!! note
+    [`--rpc-http-authentication-jwt-public-key-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-jwt-public-key-file) is only
+    required when using [JWT public key authentication](../../HowTo/Interact/APIs/Authentication.md#jwt-public-key-authentication). If
+    using [username and password authentication](../../HowTo/Interact/APIs/Authentication.md#username-and-password-authentication),
+    use [`--rpc-http-authentication-credentials-file`](../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-credentials-file) instead.
+
+You may now [start the remaining Besu nodes](Configuring-Privacy.md#7-start-besu-node-2).
```

Comment:
You may now start ... -> Start ... 

Suggested change to make more direct/simplify language

Reply:
Done